src/Servers/Kestrel/Directory.Build.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.targets))\Directory.Build.targets" />
+
+  <ItemGroup>
+    <Import Remove="System.Net.Http" />
```

Comment:
?

Reply:
There is a type name conflict between System.Net.Http.HttpMethod and Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpMethod.

Seems like this was the only conflict in our repo so I just removed it for kestrel.

Reply:
Ok, so this removes the default `using System.Net.Http`?

Reply:
Yes.

Reply:
This is an interesting problem. It finally bit in the butt.

Reply:
I'm impressed that this is the only instance.